src/commands/junit/utils.ts

```diff
@@ -1,7 +1,59 @@
+import {SpanTags} from '../../helpers/interfaces'
+import {CI_JOB_URL, CI_PIPELINE_URL, GIT_BRANCH, GIT_REPOSITORY_URL, GIT_SHA} from '../../helpers/tags'
+
 export const getBaseIntakeUrl = () => {
   if (process.env.DATADOG_SITE || process.env.DD_SITE) {
     return `https://cireport-intake.${process.env.DATADOG_SITE || process.env.DD_SITE}`
   }
 
   return 'https://cireport-intake.datadoghq.com'
 }
+
+export const getBaseUrl = () => {
+  if (process.env.DATADOG_SITE || process.env.DD_SITE) {
+    let site = process.env.DATADOG_SITE || process.env.DD_SITE
+    if (site === 'datadoghq.com') {
+      site = 'app.datadoghq.com'
+    } else if (site === 'datad0g.com') {
+      site = 'dd.datad0g.com'
+    }
+
+    return `https://${site}`
+  }
+
+  return 'https://app.datadoghq.com'
+}
+
+export const getTestRunsUrl = (spanTags: SpanTags): string => {
+  if (!spanTags[CI_PIPELINE_URL] && !spanTags[CI_JOB_URL]) {
+    return ''
+  }
+
+  let query = ''
+  if (spanTags[CI_PIPELINE_URL]) {
+    query += ` @ci.pipeline.url:"${spanTags[CI_PIPELINE_URL]}"`
+  }
+  if (spanTags[CI_JOB_URL]) {
+    query += ` @ci.job.url:"${spanTags[CI_JOB_URL]}"`
```

Comment:
do we need both query params if job url is available?

Reply:
It's not required, but I added it to quickly filter results by pipeline via removing that. I'll ask for more opinions here. If none considers it useful, I can remove it.

Reply:
got it. Yeah I'm fine keeping it

Reply:
changed!